website/webpack/prismjs-loader.js

```diff
@@ -0,0 +1,26 @@
+// modified version of https://github.com/valor-software/prismjs-loader
+'use strict';
+const Prism = require('prismjs');
```

Comment:
I don't see a prismjs dep. Even if it's implicitly available, I'd rather it be an explicit dependency.

Reply:
Come to think of it, my local build probably only worked because `prismjs` was still installed.